bot/exts/evergreen/battleship.py

```diff
@@ -140,9 +140,10 @@ def format_grid(player: Player, emojiset: EmojiSet) -> str:
     @staticmethod
     def get_square(grid: Grid, square: str) -> Square:
         """Grabs a square from a grid with an inputted key."""
-        index = ord(square[0]) - ord("A")
+        #changed to upper case
+        ind=square[0].upper()
```

Comment:
No needs for an additional variable, this can be done in-line with the next line.

Reply:
here is my discord username :𝖆𝖒𝖆𝖙𝖊𝖗𝖆𝖘𝖚#6340  ok i will write a meaningful commit message and can you help me out with this.I am new in this open source .but i cant understand whats going on